app/locale/uk.coffee

```diff
@@ -1350,7 +1350,7 @@ module.exports = nativeDescription: "Українська", englishDescription:
     required_parameters: "Обов'язкові параметри"
     optional_parameters: "Необов'якові параметри"
     returns: "Повертає"
-    granted_by: "Забезпечує"
+    granted_by: "Забезпечує" # "Забезпечено" would be better in this context: "Granted by Some Item"
```

Comment:
if you think it's more appropriate, go ahead with the change rather than adding comment!!

Reply:
I left this for me to fix later, after verifying tge context. Going to look to this on weekends.